node/resolvers/profile/index.ts

```diff
@@ -44,19 +45,33 @@ export const mutations = {
 
   subscribeNewsletter: async (
     _: any,
-    { email, isNewsletterOptIn }: SubscribeNewsletterArgs,
+    { email, name, isNewsletterOptIn }: SubscribeNewsletterArgs,
     context: Context
   ) => {
     const profile = context.clients.profile
     const optIn =
       isNewsletterOptIn === undefined || isNewsletterOptIn === true
         ? TRUE
         : FALSE
+
+    const updatedPersonalPreferences: PersonalPreferences = {
+      isNewsletterOptIn: optIn,
+    }
+
+    if (name) {
+      const userProfile = await profile.getProfileInfo({ email, userId: '' })
```

Comment:
Why `userId` is an empty string?

Reply:
Honestly, I'm not totally sure... 😅  I passed it like this because that's how it was done before